README.md

```diff
@@ -129,11 +132,15 @@ Development environment take this changes with _tutor config save_ and restart,
 
 > **Warning:** Default packages can have dependencies with other default packages or base application and disable it would break some features.
 
+:warning: **NOTE**: From Olmo version Distro has not defaulted packages. Now it is necessary to add the packages you want in ``config.yml`` file. See [How to add a new package](./docs/how_to_add_new_packages.rst)
+
 # Themes
 
 Declare the path of your themes using `tutor config save --set DISTRO_THEMES_ROOT="your_path"`,
 by default the themes path goes here **/openedx/themes**
 
+:warning: **NOTE**: From Olmo version Distro has not defaulted themes path. Now it is necessary to add the themes path in ``config.yml`` file or running command above.
```

Comment:
The same with this section of default themes. If we do not have default themes, we should drop this section. A good piece of info to put here is How to add a theme (and if you want, you can use a Bragi as example)

Reply:
Same comment here.

> You are right on this point, but I do not consider it prudent to remove this content, because we have many users using the Distro with other versions of Tutor. It is better to give some time for users to start using the Distro for this new purpose and we might update the documentation in the future.